cmd/oceanbench/main.go

```diff
@@ -168,6 +168,16 @@ var templateFuncs = template.FuncMap{
 	"localdate":     formatLocalDate,
 	"localtime":     formatLocalTime,
 	"localdatetime": formatLocalDateTime,
+	"json":          toJSON,
+}
+
+func toJSON(v interface{}) string {
```

Comment:
Please move this function into utils.go

Reply:
This could also be called "marshal", but up to you.

Reply:
I moved it to utils. True, I prefer toJSON.